ngirimana/MyDiary-Adc
# #169519869 add endpoint for adding new entry in database

> #### What does this PR do?
* add an endpoint for adding a new entry in the database
> #### Description of Task to be completed
- add new entry endpoint
- add make slug to make slug from the title
- add entry data validation
- add middleware for checking id user who is creating is registered
- add new entry route
- add test for endpoint
> #### How should this be manually tested?
* clone repository
* run ``` cd MyDiary-Adc``` in terminal
* checkout  **ft-new-entry-dbapi-169519869**  branch by running ```git checkout ft-new-entry-dbapi-169519869``` in terminal
* use 'npm install' to install all necessary dependencies
* set the environmental variable in```.env``` file
* Run 'npm run dev-start' to run the server 
* use Postman and send a post request to ```localhost:4000/api/v2/entries```
> #### Any background context you want to provide?
* you should install nodejs and Postgresql before running the repository
> #### What are the relevant pivotal tracker stories?
* [#169519869](https://www.pivotaltracker.com/story/show/169519869)
> #### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38111965/68081375-bcfd5c80-fe15-11e9-9f78-be7e7da606e8.png)
